### PR TITLE
feat: fetching bikes from services

### LINKED ISSUE #6: Get bike informations
Use services APIs to get informations on the bikes (latitude, longitude, power, range)

### PR DESCRIPTION
**Pull request by MaximeFarrugia**
fetching
close #6

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
     selected: [],
+    bikes: [],
+    refresh: false,
+  }
+
+  async componentDidUpdate() {
+    const { selected, refresh } = this.state
+    if (refresh) {
+      const data = await Promise.all(
+        selected.map(service => services[service].service(43.30, 5.36)),
```
Comment (LucasLeRay) [inline src/App.js]: Why these values ?

File: src/components/Services/LimeService.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export default function getBikes(lat, lng) {
+  return (
+    axios.get('https://web-production.lime.bike/api/rider/v1/views/map?'
+      + `ne_lat=${lat}&ne_lng=${lng}&sw_lat=${lat}&sw_lng=${lng}&user_latitude=`
+      + `${lat}&user_longitude=${lng}&zoom=16`, {
+      headers: {
+        authorization: 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.'
+          + 'eyJ1c2VyX3Rva2VuIjoiVklFT1lTTElNUkk2WiIsImxvZ2luX2NvdW50IjoxOH0.'
+          + 'wHBfoA-Wf5FGmFPdHnvR6Xuw-eWoFm4zNdMJXzuITag',
```
Comment (LucasLeRay) [inline src/components/Services/LimeService.js]: API KEY in front ?!

File: src/App.js (hunk 1 above)
Comment (MaximeFarrugia) [reply src/App.js]: I had no other values since the map is not implemented yet

File: src/components/Services/LimeService.js (hunk 2 above)
Comment (MaximeFarrugia) [reply src/components/Services/LimeService.js]: public API key

**Pull request merged**